client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -36,29 +43,14 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 		this.destroy();
 	}
 
-	// @TODO: Please update https://github.com/Automattic/wp-calypso/issues/58453 if you are refactoring away from UNSAFE_* lifecycle methods!
-	UNSAFE_componentWillReceiveProps( nextProps ) {
-		if ( this.props.isPlaying && ! nextProps.isPlaying ) {
+	componentDidUpdate( prevProps ) {
+		if ( prevProps.isPlaying && ! this.props.isPlaying ) {
 			this.pause();
-		} else if ( ! this.props.isPlaying && nextProps.isPlaying ) {
+		} else if ( ! prevProps.isPlaying && this.props.isPlaying ) {
 			this.play();
 		}
 	}
 
-	componentDidUpdate( prevProps ) {
-		if ( this.props.item.videopress_guid !== prevProps.item.videopress_guid ) {
-			this.destroy();
-		}
-	}
-
-	shouldComponentUpdate( nextProps ) {
```

Comment:
Seems like we were updating this component only if the video `guid` changes - what's the reason to remove that? Won't that cause unnecessary re-renders and/or flickers?